packages/test-scenarios/__tests__/scenario-test.js

```diff
@@ -85,69 +196,20 @@ Scenarios.fromProject(baseApp)
     project.linkDependency('babel-plugin-ember-test-metadata', {
       baseDir: __dirname,
     });
-
-    merge(project.files, {
-      tests: {
-        unit: {
-          'with-hooks-test.js': `import { module, test } from 'qunit';
-import { getTestMetadata } from '@ember/test-helpers';
-
-module('Acceptance | with-hooks-test', function (hooks) {
-  hooks.beforeEach(function () {
-    // noop
-  });
-
-  test('example', async function (assert) {
-    assert.ok(getTestMetadata(this).filePath.includes('tests/unit/with-hooks-test.js'));
-  });
-});
-`,
-          'without-hooks-test.js': `import { module, test } from 'qunit';
-import { getTestMetadata } from '@ember/test-helpers';
-
-module('Acceptance | without-hooks-test', function () {
-  hooks.beforeEach(function () {
-    // noop
-  });
-
-  test('example', async function (assert) {
-    assert.ok(getTestMetadata(this).filePath.includes('tests/unit/without-hooks-test.js'));
-  });
-});
-`,
-          'with-multiple-modules-test.js': `import {module, test} from 'qunit';
-import { getTestMetadata } from '@ember/test-helpers';
-
-module('Acceptance | with-multiple-modules-test', function (hooks) {
-  hooks.beforeEach(function () {
-    // noop
-  });
-
-  test('example', async function (assert) {
-    assert.ok(getTestMetadata(this).filePath.includes('tests/unit/with-multiple-modules-test.js'));
-  });
-});
-
-module('Acceptance | with-multiple-modules-test 2', function (hooks) {
-  hooks.beforeEach(function () {
-    // noop
-  });
-
-  test('example', async function (assert) {
-    assert.ok(getTestMetadata(this).filePath.includes('tests/unit/with-multiple-modules-test.js'));
-  });
-});
-`,
-        },
-      },
-    });
   })
   .forEachScenario((scenario) => {
     describe(scenario.name, () => {
+      const EMBROIDER = 'embroider';
+      const EMBROIDER_PATH_SEGMENT = '999999';
       let app;
+      let appTmp;
 
       beforeAll(async () => {
-        app = await scenario.prepare();
+        if (scenario.name.includes('embroider-app')) {
```

Comment:
this looks brittle. we should be able to change the scenario name without breaking this test.

Reply:
maybe we could have an enum object with fixed keys for the scenario names. e.g.

```
const SCENARIOS = {
  embroider: "embroider-app"
}
```

you would then use this lookup in both the scenario definition and here.